pkgs/development/haskell-modules/configuration-hackage2nix.yaml

```diff
@@ -2255,7 +2255,7 @@ default-package-overrides:
   - universe-instances-extended ==1.1.1
   - universe-instances-trans ==1.1
   - universe-reverse-instances ==1.1
-  - universum ==1.5.0
+  - universum ==1.6.0
```

Comment:
This list of package versions is actually generated automatically from the list of packages in Stackage's LTS-14.  So even if you change it here, it will get changed back to 1.5.0 next time the list is generated.

It might also cause a merge conflict.  Could you go ahead and change it back to 1.5.0 for this PR?

Reply:
Ok, I see - reverted back